packages/joblib/meta.yaml

```diff
@@ -1,15 +1,12 @@
 package:
   name: joblib
-  version: "0.11"
-
+  version: 1.0.1
```

Comment:
Probably better to exclude this one for now, it would likely need to update patches.

Actually the update script should not have applied on a package with patches (and the yaml is not correct)